distributed/semaphore.py

```diff
@@ -204,6 +204,25 @@ def close(self, comm=None, name=None):
 class Semaphore:
     """ Semaphore
 
+    This `semaphore <https://en.wikipedia.org/wiki/Semaphore_(programming)>`_
+    will track leases on the scheduler which can be acquired and
+    released by an instance of this class. If the maximum amount of leases are
+    already acquired, it is not possible to acquire more and the caller waits
+    until another lease has been released.
+
+    The lifetime of a lease is coupled to the ``Client`` it was acquired with.
+    Once the Client goes out of scope, the leases associated to it are freed.
+    This behavior can be controlled with the parameter
+    `distributed.scheduler.locks.lease-validation-interval`.
```

Comment:
Could you update "with the parameter `distributed.scheduler.locks.lease-validation-interval`" to "with the ``distributed.scheduler.locks.lease-validation-interval`` configuration option". Note that we should use two backticks (``) for code highlighting in sphinx